eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java

```diff
@@ -61,6 +61,7 @@ public boolean loadFromCache(InitializationCallback callback) {
                         throw new JsonSyntaxException("Configuration file missing flags");
                     }
                     flags = configResponse.getFlags();
+                    updateConfigsInSharedPrefs();
```

Comment:
We do this in the other spot we set `flags` so figured we probably want to here 🤷